### PR TITLE
[tracy] Add disable-crash-handler to disable the crash handler

### PR DESCRIPTION
**Pull request by fran6co**
Had to add this 2 commits from master:

- https://github.com/wolfpld/tracy/commit/e7ac54fba6ba52dfc92df266736f18de53df6cbc
- https://github.com/wolfpld/tracy/pull/498/commits/311ad7b0611eb87753900d5604fe09ce67549ca2

### CODE REVIEW

File: ports/tracy/vcpkg.json [hunk 1]
```diff
@@ -35,6 +35,9 @@
         }
       ]
     },
+    "disable-crash-handler": {
+      "description": "Disable crash handler"
+    },
```
Comment (dg0yt) [inline ports/tracy/vcpkg.json:40]: This feature doesn't look like "additive" functionality.
https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#features

Do you want to add a feature "crash-handler" instead?
Do users expect the crash handler to be enabled by default?

File: ports/tracy/portfile.cmake [hunk 2]
```diff
@@ -14,11 +14,13 @@ vcpkg_from_github(
...
         002-fix-capstone-5.patch
         003-fix-imgui-path.patch
+        004-fix-crash-handler.patch
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
         FEATURES
-            on-demand TRACY_ON_DEMAND)
+            on-demand TRACY_ON_DEMAND
+            disable-crash-handler TRACY_NO_CRASH_HANDLER)
```
Comment (dg0yt) [inline ports/tracy/portfile.cmake:23]: Please move the `)` to a separate line.

Comment (fran6co) [reply ports/tracy/portfile.cmake:23]: done

**Review by LilyWangLL — DISMISSED** [review]

**Comment by dg0yt** [comment]
Double-check that this PR doesn't violate the maintainer guidelines with regard to features.

**Comment by fran6co** [comment]
I could change it to `crash-handler` and make it be turned on by default to match what the normal use of the library is

**Comment by fran6co** [comment]
I just wanted it to match how the lib normally works, you would only need to care about disabling the crash handler in specific use cases

**Comment by JavierMatosD** [comment]
> I could change it to `crash-handler` and make it be turned on by default to match what the normal use of the library is

@fran6co I believe this would be the correct approach. Just make the `crash-handler` feature and have it on by default.

**Comment by fran6co** [comment]
Done

**Pull request merged**